### ISSUE TITLE
Window size/position is not remembered when opening from tray

### ISSUE THREAD

**Issue by berrnd** [canton7/SyncTrayzor#51]
Hi, and first thanks for this great wrapper for Syncthing! :)

The position and size of the main window is always reset to default when restoring from tray.
On smaller screens I use SyncTrayzor always maximized, so it would be great if the window state is saved, also when closing and restarting the application.

I have looked over the code, thought I fix it myself ... but I'm not that much into WPF ... just saw that it must have something to do with Stylets WindowManager.CreateWindow function... (?)

**Comment by canton7** [comment]
Fair request! This is a problem I know enough to know that it isn't trivial - things like monitors changing, resolution changing, etc, can cause your windows to be restored to desktops that no longer exist, or in places the mouse can't get to (like the title bar being above the top of the screen). I've got some badly-written programs which like to do things like this, and it's really annoying.

Actually implementing this will be fairly easy (I'd bind the ShellView's StartupLocation, Top, Left, Width, and Height through to properties on ShellViewModel), but I want to make sure I understand all of the gotchas before trying to implement this! Luckily I've got the whole weekend to work on SyncTrayzor, so let's see what happens.
